src/FieldWarning/Assets/Util/DontDestroyOnLoad.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DontDestroyOnLoad : MonoBehaviour
+{
+
+    public int id = 0;
+
+    // Start is called before the first frame update
+    void Awake()
```

Comment:
private void Awake

+ if id is needed, it should be called 'Id'. But I think you only increment it just before destroying the object, never reading it?

Reply:
yep its there to determine if duplicates were saved. It helps determine what objects to remove